apps/channels/views.py

```diff
@@ -90,7 +90,7 @@ def new_turn_message(request, experiment_id: uuid):
     ],
 )
 @api_view(["POST"])
-def new_api_message(request, experiment_id: uuid):
+def new_api_message(request, experiment_id: uuid, version=None):
```

Comment:
This API needs the same treatment as the other one in terms of providing separate schema for the versioned and un-versioned API.